### PR TITLE
[dev-v2.10] rancher-cis-benchmark 105.0.0+up7.0.0 unrc

### PR DESCRIPTION
**Pull request by vardhaman22**

### CODE REVIEW

File: charts/rancher-cis-benchmark-crd/7.0.0/Chart.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ apiVersion: v1
 description: Installs the CRDs for rancher-cis-benchmark.
 name: rancher-cis-benchmark-crd
 type: application
-version: 7.0.0-rc.2
+version: 7.0.0
```
Comment (pjbgf) [inline charts/rancher-cis-benchmark-crd/7.0.0/Chart.yaml]:
```suggestion
version: 105.0.0+7.0.0
```

File: charts/rancher-cis-benchmark/7.0.0/Chart.yaml [hunk 2]
```diff
@@ -12,11 +12,11 @@ annotations:
...
-appVersion: v7.0.0-rc.2
+appVersion: v7.0.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
 keywords:
 - security
 name: rancher-cis-benchmark
-version: 7.0.0-rc.2
+version: 7.0.0
```
Comment (pjbgf) [inline charts/rancher-cis-benchmark/7.0.0/Chart.yaml]:
```suggestion
version: 105.0.0+7.0.0
```

File: index.yaml [hunk 3]
```diff
@@ -9357,18 +9357,18 @@ entries:
...
-    digest: d60489eeb4de5d34679b8f9337668c52bbea7c4f03ec14108d83c362acb60700
+    digest: 11dea5161d5aeacade5ed8eb66fc4f7cd75a2b8fe70c8957d3656b3e348d7e83
     icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
     keywords:
     - security
     name: rancher-cis-benchmark
     urls:
-    - assets/rancher-cis-benchmark/rancher-cis-benchmark-7.0.0-rc.2.tgz
-    version: 7.0.0-rc.2
+    - assets/rancher-cis-benchmark/rancher-cis-benchmark-7.0.0.tgz
```
Comment (pjbgf) [inline index.yaml]:
```suggestion
    - assets/rancher-cis-benchmark/rancher-cis-benchmark-105.0.0+7.0.0.tgz
```

File: index.yaml [hunk 4]
```diff
@@ -9357,18 +9357,18 @@ entries:
...
+    version: 7.0.0
```
Comment (pjbgf) [inline index.yaml]:
```suggestion
    version: 105.0.0+7.0.0
```

File: index.yaml [hunk 5]
```diff
@@ -10023,14 +10023,14 @@ entries:
...
+    created: "2024-11-07T19:40:43.16331649+05:30"
     description: Installs the CRDs for rancher-cis-benchmark.
-    digest: d2ef89e55396bbfa91ff81b1772554d1e6b136c4238e1e4170c1d8ed1ea2da79
+    digest: cd4e6419ff1985c000e3e768ce9b13568569e383696ee7ef13357b40c405e967
     name: rancher-cis-benchmark-crd
     type: application
     urls:
-    - assets/rancher-cis-benchmark-crd/rancher-cis-benchmark-crd-7.0.0-rc.2.tgz
-    version: 7.0.0-rc.2
+    - assets/rancher-cis-benchmark-crd/rancher-cis-benchmark-crd-7.0.0.tgz
```
Comment (pjbgf) [inline index.yaml]:
```suggestion
    - assets/rancher-cis-benchmark-crd/rancher-cis-benchmark-crd-105.0.0+7.0.0.tgz
```

File: index.yaml [hunk 6]
```diff
@@ -10023,14 +10023,14 @@ entries:
...
+    version: 7.0.0
```
Comment (pjbgf) [inline index.yaml]:
```suggestion
    version: 105.0.0+7.0.0
```

File: packages/rancher-cis-benchmark/charts/Chart.yaml [hunk 7]
```diff
@@ -12,11 +12,11 @@ annotations:
...
-appVersion: v7.0.0-rc.2
+appVersion: v7.0.0
 description: The cis-operator enables running CIS benchmark security scans on a kubernetes
   cluster
 icon: https://charts.rancher.io/assets/logos/cis-kube-bench.svg
 keywords:
 - security
 name: rancher-cis-benchmark
-version: 7.0.0-rc.2
+version: 7.0.0
```
Comment (pjbgf) [inline packages/rancher-cis-benchmark/charts/Chart.yaml]:
```suggestion
version: 105.0.0+7.0.0
```

File: packages/rancher-cis-benchmark/package.yaml [hunk 8]
```diff
@@ -1,5 +1,5 @@
 url: local
-version: 7.0.0-rc.2
+version: 7.0.0
```
Comment (pjbgf) [inline packages/rancher-cis-benchmark/package.yaml]:
```suggestion
version: 105.0.0+7.0.0
```

File: release.yaml [hunk 9]
```diff
@@ -55,3 +55,7 @@ rancher-webhook:
   - 2.0.13+up0.3.13
 system-upgrade-controller:
   - 105.0.0
+rancher-cis-benchmark-crd:
+  - 7.0.0
```
Comment (pjbgf) [inline release.yaml]:
```suggestion
  - 105.0.0+7.0.0
```

File: release.yaml [hunk 10]
```diff
@@ -55,3 +55,7 @@ rancher-webhook:
...
+rancher-cis-benchmark:
+  - 7.0.0
```
Comment (pjbgf) [inline release.yaml]:
```suggestion
  - 105.0.0+7.0.0
```

**Review by pjbgf — CHANGES_REQUESTED** [review]
Versioning needs to align with #4685 for this branch.

File: charts/rancher-cis-benchmark-crd/7.0.0/Chart.yaml (hunk 1 above)
Comment (vardhaman22) [reply charts/rancher-cis-benchmark-crd/7.0.0/Chart.yaml]: done made it 105.0.0+up7.0.0

File: charts/rancher-cis-benchmark/7.0.0/Chart.yaml (hunk 2 above)
Comment (vardhaman22) [reply charts/rancher-cis-benchmark/7.0.0/Chart.yaml]: done

**Comment by vardhaman22** [comment]
> Versioning needs to align with #4685 for this branch.

done

**Review by pjbgf — APPROVED** [review]

**Comment by pjbgf** [comment]
@nicholasSUSE please review this one to make sure it will be compatible with the automation going forward.

**Comment by nicholasSUSE** [comment]
Please hold on before merging this PR, 
Let's try to execute the proper auto-bump process. 

See: https://github.com/rancher/cis-operator/pull/442

**Comment by nicholasSUSE** [comment]
Closing in favor of: https://github.com/rancher/charts/pull/4685

**Pull request closed without merging**